tests/baselines/reference/showConfig/Shows tsconfig for single option/listFilesOnly/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "compilerOptions": {
+        "listFilesOnly": true
```

Comment:
I wouldn't have guessed such a file would be created for a command-line-only option.

Reply:
But it is - it's for illustration purposes only.